main.tf

```diff
@@ -329,9 +329,10 @@ module "alb_https_sg" {
 
   name        = "${var.name}-alb-https"
   vpc_id      = local.vpc_id
-  description = "Security group with HTTPS ports open for specific IPv4 CIDR block (or everybody), egress ports are all world open"
+  description = "Security group with HTTPS ports open for specific IPv4 and IPv6 CIDR block (or everybody), egress ports are all world open"
```

Comment:
Changing the description will trigger recreation of the security group resource if I am not mistaken, right? Thinking aloud, if there are any downsides to doing this? What do you think?

Reply:
Yeah, no, good catch. Thank you. I'm reverting the change of the decription. I don't think these changes are important enough to justrify recreation of the secruity groups.